Ports/serenity-theming/package.sh

```diff
@@ -14,6 +14,7 @@ install() {
 }
 
 post_install() {
+	
```

Comment:
Why add this newline?

Reply:
I removed the comment below prior and didn't notice new line was introduced there.

Reply:
Removed newline entry on row 17.